packages/react-components/react-tree/src/index.ts

```diff
@@ -64,6 +64,7 @@ export type {
   TreeItemProps,
   TreeItemState,
   TreeItemSlots,
+  TreeItemType,
```

Comment:
you want to run `yarn workspace @fluentui/react-tree build` to regenerate api markdown

Reply:
done, thank you 👍